services/api-server/src/simcore_service_api_server/api/routes/health.py

```diff
@@ -1,8 +1,11 @@
+from datetime import datetime
+
 from fastapi import APIRouter
+from fastapi.responses import PlainTextResponse
 
 router = APIRouter()
 
 
-@router.get("/", include_in_schema=False)
+@router.get("/", include_in_schema=False, response_class=PlainTextResponse)
 async def check_service_health():
-    return ":-)"
+    return f"{__name__}@{datetime.utcnow().isoformat()}"
```

Comment:
where is the smiley now?? 🙄

Reply:
smiley got once cached and this is more useful ... :-)